exporter/zipkinexporter/zipkin.go

```diff
@@ -83,36 +83,39 @@ func createZipkinExporter(cfg *Config) (*zipkinExporter, error) {
 	return ze, nil
 }
 
-func (ze *zipkinExporter) PushTraceData(ctx context.Context, td consumerdata.TraceData) (int, error) {
-	tbatch := make([]*zipkinmodel.SpanModel, 0, len(td.Spans))
-	var resource *resourcepb.Resource = td.Resource
-
-	for _, span := range td.Spans {
-		zs, err := zipkin.OCSpanProtoToZipkin(td.Node, resource, span, ze.defaultServiceName)
-		if err != nil {
-			return len(td.Spans), consumererror.Permanent(fmt.Errorf("failed to push trace data via Zipkin exporter: %w", err))
+func (ze *zipkinExporter) PushTraceData(ctx context.Context, td pdata.Traces) (int, error) {
+	numSpans := td.SpanCount()
```

Comment:
nit: should the number of spans be extracted from the OC model (after conversion below), since you iterate over it?

Reply:
Will fix when will no longer go to OC then Zipkin. See #1446